Google.Api.Generator/ProtoUtils/ProtoExtensions.cs

```diff
@@ -50,8 +53,55 @@ public static string CSharpNamespace(this FileDescriptor desc)
                 c == '_' ? (true, acc.sb) : (false, acc.sb.Append(acc.upper ? char.ToUpperInvariant(c) : char.ToLowerInvariant(c))),
                 acc => acc.sb.ToString());
 
-        public static string CSharpName(this FieldDescriptor field) => CSharpName(field.Name, isFieldName: false);
+        public static string CSharpPropertyName(this FieldDescriptor field) => CSharpName(field.Name, isFieldName: false);
 
         public static string CSharpFieldName(this FieldDescriptor field) => CSharpName(field.Name, isFieldName: true);
+
+        private static IEnumerable<(ulong number, ByteString byteString)> GetRepeated(CustomOptions opts, int field)
+        {
+            // There is no way of reading a repeated custom option.
```

Comment:
I seem to remember thinking they never happened :) Have you filed a feature request? It may not be too hard to implement.

Reply:
That didn't occur to me ;) Yes, I'll file a request.